setup.py

```diff
@@ -99,7 +99,7 @@
     'seismology', 'seismogram', 'seismograms', 'shapefile', 'signal', 'slink',
     'spectrogram', 'StationXML', 'taper', 'taup', 'travel time', 'trigger',
     'VERCE', 'WAV', 'waveform', 'WaveServer', 'WaveServerV', 'WebDC',
-    'web service', 'Winston', 'XML-SEED', 'XSEED']
+    'web service', 'Winston', 'XML-SEED', 'XSEED', 'RG-1.6']
```

Comment:
Please keep the alphabetical order.

Reply:
There are some other ordering issues in this list, want me to fix them?